nodejs/eks/cluster.ts

```diff
@@ -127,6 +127,11 @@ export class Cluster extends pulumi.ComponentResource {
      */
     public readonly clusterSecurityGroup: aws.ec2.SecurityGroup;
 
+    /**
+     * The service role used by the EKS cluster.
+     */
+    public readonly instanceRole: ServiceRole;
```

Comment:
Minor - but any reason to expose `ServiceRole` instead of just raw `aws.iam.Role`?